jenkins/Jenkinsfile

```diff
@@ -140,6 +140,8 @@ pipeline {
             description: "The path for the bucket where the custom GCE images will be stored")
         string(name: "DATAPROC_VERSIONS", defaultValue: "1.2.79-debian9 1.4.15-debian9",
             description: "A custom image will be build for each of these dataproc versions")
+        string(name: "IMAGE_HARDENING_LIB_VERSION", defaultValue: "8277c6e",
+            description: "The hash of the 'broadinstitute/CIS-harden-images' repo branch that contains libraries for CIS-hardening GCE images")
```

Comment:
This parameter will allow us to point to newer versions of the hardening library if the AppSec team makes any updates to it.